src/main/scala/io/iohk/ethereum/network/p2p/messages/ProtocolNegotiator.scala

```diff
@@ -0,0 +1,18 @@
+package io.iohk.ethereum.network.p2p.messages
+
+import io.iohk.ethereum.network.p2p.Message.Version
+import io.iohk.ethereum.network.p2p.messages.Capability.Capabilities
+import io.iohk.ethereum.network.p2p.messages.Capability.Capabilities.Eth63Capability
+
+class ProtocolNegotiator(val protocolVersion: Version) {
+
+  val capabilities =
+    if (protocolVersion == ProtocolVersions.PV64) Capabilities.All else Seq(Eth63Capability)
+
+  // It is relatively simple now, but ETCM-355 and ETCM-356 will complicate it
```

Comment:
Why?

Reply:
I was thinking about handling etc capability separately, but probably we will stick with current impl even with PV64 and PV65 added